.travis.yml

```diff
@@ -56,8 +56,9 @@ install:
   # Install Rust.
   # TODO(ry) Include rustc in third_party.
   # https://github.com/denoland/deno/issues/386
-  if [ ! $(which rustc) ]; then
+  if [ ! $(rustc --version | grep 1.30.0) ]; then
```

Comment:
does this work when rustc isn't found at all ?

Reply:
yes